examples/config-dev.yaml

```diff
@@ -141,7 +141,7 @@ enablePasswordDB: true
 # If this option isn't chosen users may be added through the gRPC API.
 staticPasswords:
 - email: "admin@example.com"
-  # bcrypt hash of the string "password"
```

Comment:
Let's leave the original comment here. We need to clarify what the command you added does.

(This comment is valid for both files.)